extensions/labgraph_viz/setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name="labgraph_viz",
+    version="1.0.0",
+    description="Some useful visualizations for labgraph",
+    packages=find_packages(),
+    python_requires=">=3.6, <3.7",
+    install_requires=[
```

Comment:
I'd still recommend adding the other directly named imports as deps here - But this is no longer a blocker.

Reply:
Added